packages/ketcher-react/src/script/editor/utils/selectItems/finishSelecting.ts

```diff
@@ -0,0 +1,108 @@
+import { fromItemsFuse, FunctionalGroup, SGroup } from 'ketcher-core'
+import { selectElementsOnCanvas } from './selectElementsOnCanvas'
+import { isDraggingStructureOnSaltOrSolvent } from './isDraggingStructureOnSaltOrSolvent'
+import { preventSaltAndSolventsMerge } from './preventSaltAndSolventsMerge'
+
+export function finishSelecting(event, self, lassoHelper?) {
```

Comment:
code in onMouseUp in select.ts moved in finishSelecting function with little or no updates